Script/material.dat

```diff
@@ -1827,6 +1873,44 @@ liquid
     AttachedGod = CRUENTUS;
     Effect = EFFECT_MUSTARD_GAS_LIQUID;
   }
+
+  Config OMMEL_BLOOD;
```

Comment:
Please remove this material unless it forms a component in another story arc (and not the one in an IVAN fork starting with C).